Settings.cs

```diff
@@ -158,7 +158,7 @@ private void DrawJunctionsColumn()
 
         private void DrawLocoControlColumn()
         {
-            DrawColumn("DE2 Control", p => p.canControlLocomotives = GUILayout.Toggle(p.canControlLocomotives, ""));
+            DrawColumn("Locomotive Control", p => p.canControlLocomotives = GUILayout.Toggle(p.canControlLocomotives, ""));
```

Comment:
FYI this is a change in functionality.

Reply:
Not sure I understand. The label field is only passed to `GUILayout#Label`. It was changed since the DH4 can also be controlled now.

Reply:
Yeah, the DH4 being able to be controlled now is the change in functionality I'm referring to.